aflr/api_resources/mastering.py

```diff
@@ -8,17 +8,32 @@ class Mastering(APIRequest):
     def __init__(self):
         super().__init__()
         self.file_url = self.api_base + "/file/mastering"
+        self.url = self.api_base + "/mastering"
 
     def config_test(self):
         return f"Configured to transact {self.OBJECT_NAME} objects to {self.url} with api_key = {self.api_key}"
 
-	# get mastering file
+    def request(self, scriptId, backgroundTrackId, audience=[]):
+        if type(audience) != list:
+            raise Exception("audience needs to be a list.")
+        if type(backgroundTrackId) != str:
+            raise Exception("backgroundTrackId needs to be a string.")
+        if type(scriptId) != str:
+            raise Exception("scriptId needs to be a string.")
+        body = {
+            "scriptId": scriptId,
+            "backgroundTrackId": backgroundTrackId,
+            "audience": audience,
+        }
```

Comment:
@GetOn4 I dont think you need to put them into a dictionary. they are put by default. see other post requests